dev/devicelab/bin/tasks/ios_app_with_extensions_test.dart

```diff
@@ -160,34 +191,6 @@ Future<void> main() async {
         workingDirectory: flutterDirectory.path,
       );
 
-      await inDirectory(projectDir, () async {
```

Comment:
This wasn't actually building for the simulator (it's missing `--simulator`) even with the `-d` flag. `flutter build ios` with a watch companion doesn't actually work.  Anyway the next thing this does is run on the simulator, which does work.